### ISSUE TITLE
Idea: XY scatter plot for 4-5 dimensions analysis?

### ISSUE THREAD

**Issue by h-2-0** [Kozea/pygal#184]
Hi!
Is it possible to have a 4-5 dimensions analysis (X, Y, figure's size, figure' shape, text label) feature for the XY scatter plot ? I think this capability would add a lot of value to it because there are few scatter plots out there that offer this functionality.

For instance:
d1 = {'x': 12, 'y': 90, 'text': ['text string'], 'marker': {'size': 2 , 'color': 'red', 'shape':'circle}}
d2 = {'x': 2, 'y': 12, 'text': ['text string'], 'marker': {'size': 10 , 'color': 'blue', 'shape':'dot}}
d3 = ...etc.
etc.

Then, 
xy_chart.add('A', [[d1], [d2],[d3])
xy_chart.add('B', [[d4], [d5],[d6])

Thanks!

R.

tldr;

Have X, Y, figure's size, figure' shape, text label

**Comment by paradoxxxzero** [comment]
There's some options available per value : http://pygal.org/metadata/
Adding size, color and maybe shape could be nice.

**Comment by h-2-0** [comment]
Thank you, paradoxxxzero. 

I was aware of the metadata. Definitely it would be nice to have a feature like this. Having a chart that permits you analyse different dimensions I think is especially important considering this era of information overload we are living in.

**Comment by h-2-0** [comment]
I would like to comment that popular scientist Hans Rosling's scatter plot charts inspired me to ask for this feature. I think he raised the standard on how bubble charts should be used and look like (he even goes on to animate them!):

http://www.gapminder.org/world/

**Comment by paradoxxxzero** [comment]
It is possible now in the git version.
```python
  chart.add('serie', [
            {'color': 'red', 'value': 1, 'node': {'r': 12}},
            {'color': 'red', 'value': 2, 'node': {'r': 26}},
  ])
```

**Comment by h-2-0** [comment]
Thanks so much, paradoxxxzero and team!
